packages/nimble-components/src/wafer-map/tests/wafer-map.spec.ts

```diff
@@ -106,6 +125,56 @@ describe('WaferMap', () => {
         });
     });
 
+    describe('worker renderer draw flow', () => {
+        let spy: jasmine.Spy;
+        beforeEach(() => {
+            spy = spyOn(element.workerRenderer, 'drawWafer');
```

Comment:
Conventionally, we will name the `spy` local to be related to the actual member being spied on. So, here, it would be best named `drawWaferSpy`. This makes its usage in the test more readable.